src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
+
+        for (int i = 0; i < 3; i++) {
+            Lottery lottery = new Lottery();
+            System.out.println(lottery.getRandomBall());
+        }
```

Comment:
```suggestion
	Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```

we dont need to create lottery instance in the loop